### PR TITLE
[test-case-reporting] Add view for test cases, sorted by stat

### PR DESCRIPTION
**Pull request by Rafer45**
Closes https://github.com/ampproject/amp-github-apps/issues/968

### CODE REVIEW

File: test-case-reporting/app.ts [hunk 1]
```diff
@@ -128,6 +128,38 @@ app.get('/test-results/build/:buildNumber', async (req, res) => {
...
 
+app.get('/test-cases/stats/:stat', async (req, res) => {
+  const {stat} = req.params;
+  const {json, count = 10} = req.query;
+
+  try {
+    const sampleSize = Number(count);
+
+    if (Number.isNaN(sampleSize)) {
+      throw new TypeError('sampleSize is not a number');
```
Comment (rcebulko) [inline test-case-reporting/app.ts:139]:
```suggestion
      throw new TypeError(`Expected a number for parameter 'count'; got ${count}`);
```

File: test-case-reporting/static/test-case-list.html [hunk 2]
```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
```
Comment (rcebulko) [inline test-case-reporting/static/test-case-list.html:1]: Does this not use the same test-case-list template all the other views use?

Comment (Rafer45) [reply test-case-reporting/static/test-case-list.html:1]: There was no test-case-list template yet, only test-run-list and build-list templates.

File: test-case-reporting/app.ts (hunk 1 above)
Comment (Rafer45) [reply test-case-reporting/app.ts]: Done. I'll try to make my error messages more informative in the future :+1:

**Pull request merged**